Plugin/BlankCanvas/build.gradle

```diff
@@ -0,0 +1,63 @@
+// BlankCanvas Plugin build.gradle
+
+plugins {
+   id 'java-gradle-plugin'
+   id 'java'
+   id 'checkstyle'
+   id 'net.minecrell.licenser' version '0.3'
+}
+
+
+defaultTasks 'licenseFormat', 'build'
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+
+version = '1.0'
+
+repositories {
+        mavenLocal()
+        mavenCentral()
+
+        maven {
+            name = 'Sponge maven repo'
```

Comment:
You can omit the Maven repositories and`sourceCompatibility` if you use SpongeGradle (which is the recommended way to create Sponge plugins using Gradle): https://docs.spongepowered.org/master/en/plugin/project/gradle.html#using-spongegradle